lib/mocha/any_instance_method.rb

```diff
@@ -11,17 +11,12 @@ def reset_mocha
       stubbee.any_instance.reset_mocha
     end
 
-    def restore_original_method
-      return if use_prepended_module_for_stub_method?
-      if stub_method_overwrites_original_method?
-        original_method_owner.send(:define_method, method_name, original_method)
-      end
-      return unless original_visibility
-      Module.instance_method(original_visibility).bind(original_method_owner).call(method_name)
-    end
-
     private
 
+    def original_method_body
+      original_method
+    end
+
```

Comment:
Am I right in thinking this has potentially _fixed_ some un-tested behaviour in Ruby < v1.9? If so, can you explain this in the commit note?